src/components/ToDoList/ToDoList.js

```diff
@@ -33,7 +24,39 @@ export default function ToDoList({ items }) {
                             {items.filter(item => !item.completed).length}
                         </p>
                     </div>
-                    <ul className="todo-list">{listItems}</ul>
+                    <ul className="todo-list">
+                        {
+                            items.map(item =>
+                                <li key={item.id} className="todo-item">
+                                    <label>
+                                        <input
+                                            type="checkbox"
+                                            checked={item.completed}
+                                            onChange={event => toggleToDoItem(item.id, event.target.checked)}
+                                        />
+                                        {item.title}
+                                    </label>
+                                    <div className="todo-item__setting">
+
+                                        <button
```

Comment:
you use button here and Button below. Choose one

Reply:
I don't know how to make onClick prop as a optional. In some button is function, but on other not.